docker/advanced_settings.md

```diff
@@ -346,7 +346,7 @@ Notes on a couple of the parameters:
 
 * **models** - the model used when setting up service. By default it uses Squeezenet V1.1, change this argument to use customized model.
 * **worker-class** - the type of Gunicorn worker processes. We configure by default to `gevent` which is a type of async worker process. Options are described in the [Gunicorn docs](http://docs.gunicorn.org/en/stable/settings.html#worker-class).
-* **workers** - the number of Gunicorn workers which gets started. We recommend setting number of workers equal to number of vCPUs in the instance you are using. A detailed discussion of experiments and results can be found [here](../docs/optimised_config.md)
+* **workers** - the number of Gunicorn workers which gets started. We recommend setting number of workers equal to number of vCPUs in the instance you are using. A detailed discussion of experiments and results can be found [here](../docs/optimised_config.md), if it is left 'optional' MMS will automatically detect vCPUs. 
```

Comment:
Can we elaborate a bit more: "...MMS will automatically configure the number of workers to equal the number of available vCPUs in the container"